src/block_library_controller.js

```diff
@@ -152,7 +157,7 @@ BlockLibraryController.prototype.saveToBlockLibrary = function() {
   // Save block.
   this.storage.addBlock(blockType, xmlElement);
   this.storage.saveToLocalStorage();
-
+  this.initTree();
```

Comment:
For future consideration: Is this actually "init" if it is getting called in response to user actions? (fine for now)